testing/regress/ecl/fileservice.ecl

```diff
@@ -47,4 +53,8 @@ SEQUENTIAL(
   File.RenameLogicalFile(prefix + 'renametest.d00', prefix + 'afterrename1.d00', true),
   output(compareDatasets(ds, ds2)),
   File.DeleteLogicalFile(prefix + 'afterrename1.d00'),
+  RemovePrefix(File.ForeignLogicalFileName('somescope::somefilename', '192.168.168.168', true), 3);
```

Comment:
That ip looks like it won't work on most environments?

Reply:
It is a dummy IP. The function only uses it to construct a logical filename, e.g. ~foreign::192.168.168.168::blah